tests/ha/iscsi_client.pm

```diff
@@ -62,7 +63,7 @@ sub run {
     wait_still_screen 3;
     wait_serial('yast2-iscsi-client-status-0', 90) || die "'yast2 iscsi-client' didn't finish";
 
-    if (systemctl('-q is-active iscsi', ignore_failure => 1)) {
+    if (is_sle('>=15') && systemctl('-q is-active iscsi', ignore_failure => 1)) {
```

Comment:
Ok, but in that case you should restrict to 15-SP2, as we had no issue on 15 and 15-SP1, so better to do this for now: `is_sle('=15-SP2')`, as a bug is open for this issue, and when this will be fixed this workaround will be removed.

Reply:
> Ok, but in that case you should restrict to 15-SP2, as we had no issue on 15 and 15-SP1, so better to do this for now: `is_sle('=15-SP2')`, as a bug is open for this issue, and when this will be fixed this workaround will be removed.

I agree, thanks for pointing me that.